lib/iex/lib/iex/introspection.ex

```diff
@@ -745,9 +749,11 @@ defmodule IEx.Introspection do
     end
   end
 
-  defp elixir_module?(module) do
-    function_exported?(module, :__info__, 1)
-  end
+  defp translate_doc(nil), do: nil
+  defp translate_doc(:none), do: nil
+  defp translate_doc(:hidden), do: nil
+  defp translate_doc(binary) when is_binary(binary), do: binary
```

Comment:
When can we hit this clause?

Reply:
From [line 379](https://github.com/lackac/elixir/blob/a59baf22465a279edc88c477b9e19e83e5d9cae9/lib/iex/lib/iex/introspection.ex#L379)

Reply:
Similarly, if we can avoid `nil` and use `:none` instead... but I am not sure how feasible is that.

Reply:
This is exactly the symptom. Currently the code is in two minds. Do we keep the atoms used by the chunk and let clients deal with that or translate them early on into idiomatic Elixir? I'm not sure which I would prefer, but certainly not both. :)

The current implementation was mostly a mechanical replacement of `Code.get_docs/1`. I think with spending more time on it we could refactor the IEx related bits more. Moving some things like `count_defaults` to the chunk while sharing some code between `IEx.Autocomplete` and `IEx.Introspection`.